src/css/index.css

```diff
@@ -0,0 +1,218 @@
+body
+{
+    margin: 0px;
+    background-color: var(--theme-ui-colors-background,#0C0717);
+    color: var(--theme-ui-colors-text,white);
+    box-sizing: border-box;
+}
+.heading
+{
+    box-sizing: border-box;
```

Comment:
Instead of using box-sizing: border-box in every component, you can use the * (all) selector
`
\  * {
     box-sizing: border-box;
}
`